skins/oasis/modules/ContributeMenuController.class.php

```diff
@@ -36,6 +36,15 @@ public function executeIndex() {
 				'href' =>  $specialPageTitle->getLocalURL(),
 			];
 
+			/**
+			 * id for flow tracking
+			 * @see https://wikia-inc.atlassian.net/browse/WW-343
+			 */
+			if ( $specialPageName === 'CreatePage' ) {
```

Comment:
Did you consider adding new hook to update dropdown items or adding `id` and additional url param here https://github.com/Wikia/app/blob/af465d11aa798eeef022e6002c59a81e99a96d54/skins/oasis/modules/ContributeMenuController.class.php#L11 if FlowTracking extension is enabled?